dissect/target/plugin.py

```diff
@@ -384,6 +384,18 @@ def os(self) -> str:
         """
         raise NotImplementedError
 
+    @export(property=True)
+    def architecture(self) -> Optional[str]:
+        """Required OS function.
+
+        Implementations must be decorated with ``@export(property=True)``
+
+        Returns:
+            A slug of the OS architecture, e.g. 'x86_32-unix', 'MIPS-linux' or
```

Comment:
```suggestion
            A slug of the OS architecture, e.g. 'x86_32-unix', 'MIPS-linux' or
```
Should we standardize this to target triplets? https://wiki.osdev.org/Target_Triplet

Reply:
That would be nice yes.